scripts/reproduce-genesis-locally.bash

```
@@ -0,0 +1,31 @@
+FLUENTBASE_BUILD_DOCKER_IMAGE=fluentbase-build \
+FLUENTBASE_BUILD_DOCKER_TAG=local \
+FLUENTBASE_CONTRACTS_IGNORE_DEFAULT_RUST_FLAGS=true \
+cargo b --release
+
+TMP_DIR="$(mktemp -d)"
+BASE_URL="https://github.com/fluentlabs-xyz/fluentbase/releases/download/${TAG}"
+
+curl -fL "$BASE_URL/genesis-mainnet-${TAG}.json.gz" -o "$TMP_DIR/remote-mainnet.json.gz"
+gunzip -c "$TMP_DIR/remote-mainnet.json.gz" > "$TMP_DIR/remote-mainnet.json"
+
+REMOTE_MAINNET_SHA="$(sha256sum $TMP_DIR/remote-mainnet.json | awk '{print $1}')"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)